File: abc/WhenIsWeekend/Lambda.cpp
#include <ctime>
#include <iostream>

using namespace std;

int main() {
    cout << "When’s Weekend?" << endl;

    time_t t = time(nullptr);
    int today = localtime(&t)->tm_wday;

    switch (today) {
        case 0:
            cout << "Today." << endl;
            break;
        case 6:
            cout << "Today." << endl;
            break;
        case 5:
            cout << "Tomorrow." << endl;
            break;
        case 4:
            cout << "In two days." << endl;
            break;
        default:
            cout << "Too far away." << endl;
    }
}

File: abc/WhenIsWeekend/Lambda.py
import time

def main() -> None:
    print("When’s Weekend?")

    today = time.localtime().tm_wday

    if today == 6:
        print("Today.")
    elif today == 5:
        print("Today.")
    elif today == 4:
        print("Tomorrow.")
    elif today == 3:
        print("In two days.")
    else:
        print("Too far away.")

if __name__ == "__main__":
    main()
